### DISCUSSION TITLE
Any option to exclude dependency with FastDepends from docs?

### DISCUSSION THREAD

**Discussion by shepilov-vladislav** [ag2ai/faststream#928] [Q&A]
Any option to exclude dependency with FastDepends from docs?

**Comment by Lancetnik** [comment]
Only by schema.json manual customization and serving from the file.
Can you show me an example you want? Dependencies were designed to be a part of an incoming message, so they should be represented in the schema too from my point of view

**Reply by shepilov-vladislav** [reply]
For example I want to add a repository pattern as a dependency, in order to save changes to the base after receiving a message. The repository class itself is not a pydantic class and so I get an error when I try to serve the documentation `PydanticInvalidForJsonSchema: Cannot generate a JsonSchema for core_schema.IsInstanceSchema (<class 'app.infrastructure.database.repository.SQLAlchemyRepository'>)`.

**Reply by shepilov-vladislav** [reply]
@Lancetnik `FastAPI` for example does not show such dependencies in OpenAPI because they are internal. This behavior seems more logical to me, what do you think?
